opencti-platform/opencti-front/lang/back/en.json

```diff
@@ -181,6 +182,7 @@
   "Disabled": "Disabled",
   "Display name": "Display name",
   "DLL service": "DLL service",
+  "Do you want to delete this support package?": "Do you want to delete this support package?",
```

Comment:
Why this translation is in `back` folder? Should be only on `front` folder I guess